### ISSUE TITLE
Request A & D keys for movement ASAP!!!

### ISSUE THREAD

**Issue by Artas1984** [nipkownix/re4_tweaks#521]
From what i understood this mod has been for a while now. I thank you for creating it, for it basically made RE4 interface and gameplay from outdated "wooden" to modern and "elastic", but it is absolutely shocking to me that the one MOST IMPORTANT feature is missing: assigning A and D keys for direct movement sideways, respectively LEFT and RIGHT! On default keys A and B just move the camera left and right with character standing in place like an idiot, but in literally every other game in history that i have played, these keys MOVE your character in A DIRECTION...

After all these years that the tweak mod exists, it is hard to believe that keys A & D are still for camera only... I 've seen famous youtubers complaining about this as well in their walkthroughs of Resident Evil 4.

At least this mod enables player rotation by mouse (RE5 style), which was a blessing, but that alone is not enough. 

Sincerely, please add this feature ASAP!!!

**Comment by LethalPlacebo** [comment]
This is being worked on in a different fork of RE4 Tweaks that is being managed by someone else. You'll have to build it yourself using the instructions on the page. I believe they posted a download in one of the issues here, but I have no idea where that is:

https://github.com/aman-11111/re4_tweaks

**Comment by Artas1984** [comment]
> This is being worked on in a different fork of RE4 Tweaks that is being managed by someone else. You'll have to build it yourself using the instructions on the page. I believe they posted a download in one of the issues here, but I have no idea where that is:
> 
> https://github.com/aman-11111/re4_tweaks

Sorry, i definitely ain't gonna ''BUILD'' anything myself, i for i do not understand any of this scripting stuff, i just need a finished product. Speaking of the link you provided - i do not see any ''experimental'' dinput8.dll and dinput8.ini files. 

Can somebody point me DIRECTLY to those files?

**Comment by LethalPlacebo** [comment]
> > This is being worked on in a different fork of RE4 Tweaks that is being managed by someone else. You'll have to build it yourself using the instructions on the page. I believe they posted a download in one of the issues here, but I have no idea where that is:
> > https://github.com/aman-11111/re4_tweaks
> 
> Sorry, i definitely ain't gonna ''BUILD'' anything myself, i for i do not understand any of this scripting stuff, i just need a finished product. Speaking of the link you provided - i do not see any ''experimental'' dinput8.dll and dinput8.ini files.
> 
> Can somebody point me DIRECTLY to those files?

Yeah, as I pointed out, that link is to the fork where you can build it yourself. In one of the issues threads here on the main branch, that person posted their progress at one point in a direct link, but I dunno where it is anymore.
